Revise targeting docs (#570)
configcat/docs: website/docs/targeting/user-object.mdx

### Identifier attribute

> The unique identifier of a user in your application. This is a **REQUIRED** attribute, which enables ConfigCat to differentiate your users from each other and to evaluate Targeting Rules and Percentage Options.

Author: ```suggestion
The unique identifier of a user in your application. This is a **REQUIRED** attribute. ConfigCat uses this attribute to differentiate your users from each other. By default, it is used to evaluate Percentage Options (link).
```

Our recommendation for an identifier:

> - User ID - If such an ID is available in your application, you can as well use it with ConfigCat.

Author: ```suggestion
- User ID - You can simply use the user ID native to your application.
```

Jó lenne majd beleírni, hogy mik a veszélyei ennek.
